scripts/prepare_transport_data_input.py

```diff
@@ -12,56 +12,87 @@
 import pandas as pd
 from _helpers import BASE_DIR
 
-# logger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
+
+
+def _add_iso2_code_per_country_and_clean_data(df):
+
```

Comment:
Would you mind adding a minor string or drop this line?

Reply:
Sure, added a minor doc string.